core/src/main/java/org/apache/iceberg/hadoop/HadoopFileIO.java

```diff
@@ -49,6 +62,11 @@ public class HadoopFileIO implements HadoopConfigurable, DelegateFileIO {
 
   private static final Logger LOG = LoggerFactory.getLogger(HadoopFileIO.class);
   private static final String DELETE_FILE_PARALLELISM = "iceberg.hadoop.delete-file-parallelism";
+
+  /** Is bulk delete enabled on hadoop runtimes with API support: {@value}. */
+  public static final String BULK_DELETE_ENABLED = "iceberg.hadoop.bulk.delete.enabled";
+
+  private static final boolean BULK_DELETE_ENABLED_DEFAULT = false;
```

Comment:
Is there a reason we wouldn't have this on by default?  Is there a reason we would even need this to be an option?  If this is available in the linked version of hadoop, I would prefer just removing the options and making it always on.